indico/modules/events/abstracts/templates/management/_notification_tpl_list.html

```diff
@@ -0,0 +1,76 @@
+{% from 'message_box.html' import message_box %}
+
+{% macro render_notification_list(event, email_tpls) -%}
+
+    {% if email_tpls|list|length == 0 %}
```

Comment:
`not email_tpls`? and IMO this should already be something list-like so no need for `|list` (passing query objects to templates is not a good idea anyway in case that's why you needed `|list`)